### PR TITLE
[release/v2.5.9] Releasing chart for 2.5.9

### PR DESCRIPTION
**Pull request by jiaqiluo**
```
# Assuming that your upstream remote (e.g. https://github.com/rancher/charts.git) is named `upstream` 
# Replace the following environment variables
STAGING_BRANCH=dev-v2.5.9
LIVE_BRANCH=release-v2.5
FORKED_BRANCH=new-charts-for-259
git fetch upstream
git checkout upstream/${LIVE_BRANCH} -b ${FORKED_BRANCH}
git branch -u origin/${FORKED_BRANCH}
git checkout upstream/${STAGING_BRANCH} -- charts assets index.yaml
git add charts assets index.yaml
git commit -m "Releasing chart"
git push --set-upstream origin ${FORKED_BRANCH}
# Create your pull request!
```

the issue https://github.com/rancher/rancher/issues/33557

### CODE REVIEW

File: index.yaml [hunk 1]
```diff
@@ -3158,6 +3158,21 @@ entries:
...
+      catalog.cattle.io/release-name: rancher-webhook
+    apiVersion: v2
+    appVersion: 0.1.1
+    created: "2021-07-12T10:25:52.205998-07:00"
+    description: ValidatingAdmissionWebhook for Rancher types
+    digest: a0ee1c4f1338c8c24f3c5129bb41a67a43bffb13bf28c138266cf58dd28f2ce4
+    name: rancher-webhook
+    urls:
+    - assets/rancher-webhook/rancher-webhook-0.1.100+up0.1.1.tgz
+    version: 0.1.100+up0.1.1
```
Comment (jiaqiluo) [inline index.yaml:3175]: the only new entry

**Review by aiyengar2 — APPROVED** [review]
LGTM from a charts perspective. Need a second review for the contents of rancher-webhook to make sure the necessary changes have gone in.

**Review by ryansann — APPROVED** [review]

File: charts/rancher-webhook/rancher-webhook/0.1.100+up0.1.1/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/namespace: cattle-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/release-name: rancher-webhook
+apiVersion: v2
+appVersion: 0.1.1
+description: ValidatingAdmissionWebhook for Rancher types
+name: rancher-webhook
+version: 0.1.100+up0.1.1
```
Comment (cbron) [inline charts/rancher-webhook/rancher-webhook/0.1.100+up0.1.1/Chart.yaml:11]: Doesn't need a min rancher version as its being introduced here to fix former rancher versions. 
Last version was 0.1.000 so 0.1.100 is correct.

**Review by cbron — APPROVED** [review]
lgtm, waiting to merge until system-charts is ready also

**Pull request merged**